openpower/configs/hostboot/witherspoon.config

```diff
@@ -1,48 +1,46 @@
 # The Serial Flash Controller is the AST2400 BMC.
-set   SFC_IS_AST2400
+set   SFC_IS_AST2500
+unset SFC_IS_AST2400
```

Comment:
The default comes from the HB repo though.  So are we going to hold off on this change until the HB change gets merged through?